config/env/all.js

```diff
@@ -5,5 +5,5 @@ var keys = rootPath + '/keys.txt';
 module.exports = {
 	root: rootPath,
 	port: process.env.PORT || 3000,
-    db: process.env.MONGOHQ_URL
+    db: process.env.MONGOHQ_URL || 'mongodb://localhost:27017'
```

Comment:
What I see you're doing is specifying the MongoDB URL for the database as `mongodb://localhost:27017` if it doesn't exist in the environment variables.

I think you could specify this URL in the `MONGOHQ_URL` environment variable for this project.

Checkout the `dotenv` module.